src/main/java/uk/nhs/tis/sync/dto/PersonDmsDto.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.tis.sync.dto;
+
+import lombok.Data;
+
+/**
+ * A DTO representing the expected structure for Person via DMS.
+ */
+@Data
+public class PersonDmsDto {
+
+  private final String id;
+  private final String intrepidId;
+  private final String addedDate;
+  private final String amendedDate;
```

Comment:
Is there a reason we include amendedDate here, but not for e.g. PlacementSpecialty? I don't think we use it anywhere downstream (it doesn't make it into the TraineeProfile) - which is not to say its not useful, but I'm wondering if we need to include this field in any of the other DmsDtos?

Reply:
The fields match those in the database table, while we don't use most of the fields I wanted to imitate the data we get from DMS as closely as possible.
There's definitely an argument for trimming these DTOs right back to only what TSS needs, but I don't think we should do that until (if) we pull all of this in to it's own data request service

Reply:
As for why there are fields missing from PlacementSpecialty, it's either an oversight or they weren't available. I can't see a valid reason for it, but I didn't have much involvement in that initial work.

Reply:
Ah, it was that Marcello cowboy! :trollface: 

I agree with keeping the fields as similar to the data from DMS as possible at this point. I guess that Reuben cowboy should add amendedDate to the CurriculumMembership and Qualification DmsDtos now 💥